server/src/main/java/org/cloudfoundry/identity/uaa/web/beans/UaaDefaultCookieSerializer.java

```diff
@@ -0,0 +1,434 @@
+package org.cloudfoundry.identity.uaa.web.beans;
```

Comment:
Do you need license headers here?

Reply:
UAA and spring use the same apache license: "Licensed under the Apache License, Version 2.0 (the "License");"
We don't provide any additional license headers for predix customizations; keep Pivotal headers wherever present.
Based on that decided not to add a header